Modules/main.c

```diff
@@ -1398,6 +1398,7 @@ pymain_read_conf(_PyMain *pymain, _PyCoreConfig *config,
 void
 _PyMainInterpreterConfig_Clear(_PyMainInterpreterConfig *config)
 {
+    config->install_signal_handlers = -1;
```

Comment:
I don't think that it's needed, this function is only used to release memory, not to reset a config to the default config.

If you really need to reset to the default config, I would prefer to use _PyMainInterpreterConfig_INIT to reduce maintenance (avoid duplication of code).